### PR TITLE
[FEATURE] Link site.websiteTitle and PageTitle API

### PR DESCRIPTION
**Pull request by cweiske**
So that it's easier to find other relevant settings.

### CODE REVIEW

File: Documentation/ApiOverview/Seo/PageTitleApi.rst [hunk 1]
```diff
@@ -147,3 +147,9 @@ First the :php:`SeoTitlePageTitleProvider` (because it will be handled before
 You can override these settings within your own installation. You can add as
 many providers as you want. Be aware that if a provider returns a non-empty
 value, all provider with a lower priority will not be checked.
+
+
+See also
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Seo/PageTitleApi.rst:152]: We have also a `.. seealso::` admonition, this may be better suited. What do you think?

**Review by brotkrueml — COMMENTED** [review]
Good idea, I love cross-references :+1:

**Comment by cweiske** [comment]
I changed it to the admonition, but moved it up because when it's inline it can't be a part of priority section.

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@cweiske Thanks for the improvement :+1:

**Pull request merged**